content/pages/astronomers/applications.md

```diff
@@ -0,0 +1,108 @@
+---
+title: "VO Applications"
+date: 2024-02-09T10:00:00+02:00
```

Comment:
The `date` can be left out - this is not correct anyways and Hugo will insert it automatically.